### PR TITLE
Support multi vehicle simulation for fixed wing in gazebo SITL

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
**Describe problem solved by this pull request**
Since the plane model is defined as a sdf format, it was not possible to launch multivehicle simulation, as it relies on xacro macros to generate sdf files.

**Describe your solution**
This enables https://github.com/PX4/sitl_gazebo/pull/385 and spawns the vehicles from the sdf file rather than froma urdf file as done previously. This is more consistent on what was being done for the iris in PX4/sitl_gazebo

**Test data / coverage**
Video: 
[Video](https://youtu.be/aEzFKPMEfjc "Hovering done")

### CODE REVIEW

File: Tools/gazebo_sitl_multiple_run.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ done
 num_vehicles=${NUM_VEHICLES:=3}
 export PX4_SIM_MODEL=${VEHICLE_MODEL:=iris}
 
-if [ "$PX4_SIM_MODEL" != "iris" ]
+if [ "$PX4_SIM_MODEL" != "iris" ] & [ "$PX4_SIM_MODEL" != "plane" ]
 then
 	echo "Currently only iris vehicle model is supported!"
```
Comment (bkueng) [inline Tools/gazebo_sitl_multiple_run.sh]: This needs to be updated.

Comment (RomanBapst) [reply Tools/gazebo_sitl_multiple_run.sh]: @Jaeyoung-Lim I think you also need to update the comment :-)

**Comment by Jaeyoung-Lim** [comment]
@RomanBapst Thanks!

**Comment by karanveerdewan** [comment]
Hi folks,

I am facing an issue while using multi-vehicle SITL with Gazebo. The "iris" model works flawlessly even with 10 instances following a mission via QGCS (real time factor close to 1). But while running "plane" or "standard_vtol" model, even when they are grounded, the simulation speed real time factor drops down to ~0.1 (even at 2 instances), even lesser when adding more instances.

How do I make sure the "plane" model (multiple instances) also works at a real time factor close to 1?

TIA

**Comment by Jaeyoung-Lim** [comment]
@karanveerdewan Please don't hijack the topic of this PR. If you have questions post a separate issue or post the question on discuss.px4.io

**Comment by Jaeyoung-Lim** [comment]
@SalimTerryLi Please post a new issue on how to reproduce your issue. Following the steps on myside, it works so you need to provide exactly what you are doing so that I can reproduce your issue.

This PR is not related to your issue, so please post a new issue

**Pull request merged**